fleetspeak/src/common/proto/fleetspeak_monitoring/resource.proto

```diff
@@ -27,8 +29,11 @@ message ResourceUsageData {
 
   int64 pid = 2;
 
+  // The self reported service/service binary version.
+  string version = 3;
```

Comment:
I think you mean to set this as field number 8, no?

Reply:
done